spec/acceptance/pkcs12_spec.rb

```diff
@@ -144,29 +134,29 @@
         }
       MANIFEST
 
-      idempotent_apply(default, pp)
+      idempotent_apply(pp)
     end
 
     expectations = [
       %r{Alias name: leaf_cert},
       %r{Entry type: (keyEntry|PrivateKeyEntry)},
       %r{Certificate chain length: 3},
-      %r{^Serial number: 5$.*^Serial number: 4$.*^Serial number: 3$}m,
+      %r{Serial number: 5},
+      %r{Serial number: 4},
+      %r{Serial number: 3},
     ]
     it 'verifies the private key and chain' do
-      shell("\"#{@keytool_path}keytool\" -list -v -keystore #{target} -storepass puppet") do |r|
-        expect(r.exit_code).to be_zero
+      run_shell((keystore_list target), expect_failures: true) do |r|
+        expect(r.exit_code).to eq(@exit_code)
         expectations.each do |expect|
           expect(r.stdout).to match(expect)
         end
       end
     end
-    # -keypasswd commands not supported if -storetype is PKCS12 on ubuntu 18.04 with current java version
-    unless os[:family] == 'ubuntu' && os[:release].start_with?('18.04')
```

Comment:
This restriction would appear to longer be true. These tests run fine against 18.04